controllers/books.js

```diff
@@ -0,0 +1,9 @@
+const Books = (connection, Sequelize, Authors) => {
+  return connection.define('books', {
+    id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+    title: { type: Sequelize.STRING, allowNull: false },
```

Comment:
What made you want to change the name from novels to books? Something again that should be changed in your documentation as novels and novelsGenres is what had been the documented names and routes